.github/workflows/check-pull-request.yaml

```
@@ -30,11 +30,55 @@ jobs:
     steps:
     - name: Checkout the source
       uses: actions/checkout@v2
+      with:
+        persist-credentials: false
 
     - name: Setup Go
       uses: actions/setup-go@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)